src/openms/source/FORMAT/HANDLERS/IndexedMzMLHandler.cpp

```diff
@@ -247,7 +247,6 @@ namespace OpenMS::Internal
   {
     OpenMS::MSSpectrum s;
     getMSSpectrumById(id, s);
-    s.updateRanges();
```

Comment:
maybe leave a comment why not to update ranges for future readers?

Reply:
this was just a double update